backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v2/externlasystem/ExternalSystemRestController.java

```diff
@@ -202,6 +203,21 @@ public ExternalSystemRestController(final ExternalSystemService externalSystemSe
 		return ResponseEntity.ok().body(statusInfo);
 	}
 
+	@GetMapping(path = "/{externalSystemConfigType}/{externalSystemChildConfigValue}/info")
```

Comment:
Please add a docu saying that right now just GRS is supported, to keep API users from guessing

Reply:
..or did i misread and it works for any config type?